app/models.py

```diff
@@ -2263,7 +2263,7 @@ def serialize(self):
             'template_name': self.template.name,
 
             'personalisation': self.personalisation,
-            'areas': self.areas,
+            'areas': self.areas.get("areas", []),
```

Comment:
Should we also expose `simple_polygons` too in a similar way, for example

```
'simple_poloygons': self.areas.get("simple_polygons", []),
```

Reply:
good point, @servingUpAces what do you think?

Reply:
I added it, sounds like a sensible suggestion.